mdx-layout-core/src/CoreLayout.tsx

```diff
@@ -1,8 +1,26 @@
 import type { Context } from '@divriots/studio-doc-compiler';
 import { PageGraph, buildGraph } from './PageGraph';
-import './CoreLayout.scss';
 import { NavBar } from './NavBar';
-import React, { ReactChildren, ReactChild } from 'react';
+import React, { ReactChildren, ReactChild, CSSProperties } from 'react';
+
+import 'https://cdnjs.cloudflare.com/ajax/libs/prism/1.23.0/themes/prism-tomorrow.min.css';
+
+const styles = {
+  main: {
+    display: 'grid',
+    justifyContent: 'stretch',
+  } as CSSProperties,
+
+  article: {
+    marginLeft: 'auto',
+    marginRight: 'auto',
+    overflow: 'auto',
+    boxSizing: 'border-box',
+    width: '100%',
+    maxWidth: '800px',
+    padding: '0 1rem',
+  } as CSSProperties,
+};
```

Comment:
now when I see you meant inlined styles in the context of CSS in JS, I can say a few more things
as a workaround for now it's OK, I think you want to fix production issue now, so it's OK

but in general inlines styles is the worst type of CSS in JS because of DX when debugging and slowest performance possible, especially for dynamic styles
we should better use CSS Modules I think or some sort of CSS in JS which compiles everything statically in the build and doesn't require a runtime

Reply:
Definitely agree! I tried to make the fix quick and clear enough, without adding more dependencies. But we need to reshape our strategy for the dockit layout system, probably based-on CSS Module, or Vanilla-extract, or styled-component. But it's for another PR :D